### PR TITLE
Fix category checkbox redirect (Backport #16159)

### LINKED ISSUE #16119: When I want to check a category, it's like I clicked on it instead of just select it
**Describe the bug**
When I want to check a category, it's like I clicked on it instead of just select it

**To Reproduce**
1. Go to Catalog > Categories
2. Try to check a checkbox
3. You are redirected like you have clicked on it

**Screenshots**
https://drive.google.com/open?id=1p89GeJ8XKjNZPewkjeUhZ_YLW_rOEJqK

**Additional information**
PrestaShop version: develop
PHP version: N/A

### PR DESCRIPTION
**Pull request by atomiix**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Prevent click action on identifier type table cells that have a checkbox 
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #16119
| How to test?  | see #16119

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16265)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

**Review by matthieu-rolland — DISMISSED** [review]
looks fine to me

File: admin-dev/themes/new-theme/js/components/grid/extension/link-row-action-extension.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ export default class LinkRowActionExtension {
          * Only search for cells with non clickable contents to avoid conflicts with
          * previous cell behaviour (action, toggle, ...)
          */
-        const clickableCells = $('td.data-type, td.identifier-type, td.badge-type, td.position-type', $parentRow)
+        const clickableCells = $('td.data-type, td.identifier-type:not(:has(.md-checkbox)), td.badge-type, td.position-type', $parentRow)
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/js/components/grid/extension/link-row-action-extension.js]: What about radio? text? etc :thinking: Are we sure, `.md-checkbox` is enough? what if a thrid part module use a custom css class?

Could we test something more generic?
```suggestion
        const clickableCells = $('td.data-type, td.identifier-type:not(:has(input)), td.badge-type, td.position-type', $parentRow)
```

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Forget to add this comment :roll_eyes:

**Review by PierreRambaud — APPROVED** [review]

**Comment by matks** [comment]
Thank you @atomiix

**Comment by eternoendless** [comment]
[Tests are green](https://travis-ci.com/PrestaShop/PrestaShop/builds/135001226) but not reported to the PR, merging.

**Pull request merged**